src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/activities/ActivitiesService.kt

```diff
@@ -76,6 +77,43 @@ class ActivitiesService(
     }
   }
 
+  fun createAllocation(allocationEvent: AllocationDomainEvent) {
+    activitiesApiService.getActivitySchedule(allocationEvent.scheduleId).let { schedule ->
```

Comment:
yes possibly - I was using schedule start/end dates at one point. Ill leave until its been integration tested - something else might be needed.

Reply:
But what would be needed? Surely we already have the schedule in NOMIS by this time, I'm struggling to think of data we need to get about the schedule that we haven't already synchronised - unless of course the new service models the Schedule radically different, and bits that were on the allocation are now in the schedule (which is more than possible!!).

But I suspect we should start simple and complicate after we find that out rather than start complicated and simplify.

Reply:
ok, will remove schedule call